repository/src/main/resources/alfresco/subsystems/email/OutboundSMTP/mail-template-services-context.xml

```diff
@@ -161,4 +161,25 @@
             <ref bean="ServiceRegistry"/>
         </property>
     </bean>
+
+    <bean id="testDocumentUrl" parent="mailBaseTemplateImplementation" class="org.alfresco.repo.template.DocumentationURLMethod">

```

Comment:
Why this bean id is `testDocumentUrl`? Can we change id to e.g. `documentUrlExtenstion`?